src/Admin/PostAdmin.php

```diff
@@ -48,26 +45,29 @@ class PostAdmin extends AbstractAdmin
      * @var PermalinkInterface
      */
     protected $permalinkGenerator;
-
     /**
-     * @deprecated since sonata-project/news-bundle 3.13, to be removed in 4.0.
+     * @var SonataConfiguration
      */
-    public function setUserManager($userManager): void
+    private $sonataConfiguration;
+
+    public function __construct(string $code, string $class, string $baseControllerName, SonataConfiguration $sonataConfiguration)
```

Comment:
I wonder if you have something better in mind than injecting the whole configuration @franmomu

Reply:
nope 😞 I think this option is not in the container as parameter.